docker/docker-compose.dev.yml

```diff
@@ -6,6 +6,7 @@ services:
     build:
       context: ..
       dockerfile: ./docker/Dockerfile.dev
+    command: python /code/manage.py runserver -h 0.0.0.0 -p 80
```

Comment:
curious, why did you move this?

Reply:
@mayhem, I was looking at the Dockerfile to see why debug messages didn't seem to be getting printed, so I tried moving the command here. Anyways, doing this in the compose file looked better to me (and it is consistent with how we do it in LB and MsB etc.)